src/estimagic/algorithms.py

```diff
@@ -0,0 +1,46 @@
+import inspect
+
+from estimagic.optimization import (
+    bhhh,
+    cyipopt_optimizers,
+    fides_optimizers,
+    nag_optimizers,
+    neldermead,
+    nlopt_optimizers,
+    pounders,
+    pygmo_optimizers,
+    scipy_optimizers,
+    simopt_optimizers,
+    tao_optimizers,
+    tranquilo,
+)
+
+MODULES = [
+    cyipopt_optimizers,
+    fides_optimizers,
+    nag_optimizers,
+    nlopt_optimizers,
+    pygmo_optimizers,
+    scipy_optimizers,
+    simopt_optimizers,
+    tao_optimizers,
+    bhhh,
+    neldermead,
+    pounders,
+    tranquilo,
```

Comment:
Why not stick to the import order? It seems less error-prone when we update this.

```suggestion
    bhhh,
    cyipopt_optimizers,
    fides_optimizers,
    nag_optimizers,
    neldermead,
    nlopt_optimizers,
    pounders,
    pygmo_optimizers,
    scipy_optimizers,
    simopt_optimizers,
    tao_optimizers,
    tranquilo,
```

Reply:
Will do that next time.